bootstrap/cli.py

```diff
@@ -0,0 +1,141 @@
+import os
+
+import inquirer
+
+from minifi_option import MinifiOptions
+from system_dependency import install_required
+
+
+def install_dependencies(minifi_options: MinifiOptions):
+    install_required(minifi_options)
+
+
+def run_cmake(minifi_options: MinifiOptions):
+    if not os.path.exists(minifi_options.build_dir):
+        os.mkdir(minifi_options.build_dir)
+    os.chdir(minifi_options.build_dir)
+    cmake_cmd = f"cmake -G Ninja {minifi_options.create_cmake_options_str()} {minifi_options.source_dir}"
```

Comment:
This could default to `FORCE_COLORED_OUTPUT=ON`, since Ninja buffers the output instead of just letting the subprocesses write to the terminal directly, resulting in no color.

Reply:
We could do that. Automations could still explicitly set it to off if needed. We should mention this in the readme if we change the default.

Reply:
Changed the default, https://github.com/apache/nifi-minifi-cpp/pull/1681/commits/0e43424c2a2f2567876534b06fa68ef005104874
But im not sure where (if at all) we should include this in the readme